### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by Nkaleth**
Objectives In this PR: 

- Remove all hardcoded items from the tasks array.
- Create a new JavaScript file for the new functionality.
- Implement a function for adding a new task (add a new element to the array).
- Implement a function for deleting a task (remove an element from the array).
- Implement a function for editing task descriptions.
- By default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).
- Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1)..
- All changes to the To Do List should be saved in local storage.

### CODE REVIEW

File: src/modules/Tasks.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  static removeObWithId = (arr, id) => {
+    const ObToRemove = arr.findIndex((obj) => obj.index === id);
+    if (ObToRemove > -1) {
+      arr.splice(ObToRemove, 1);
+    }
+    return arr;
+  };
+
+  static renewIndex = (arr) => {
+    let index = 0;
```
Comment (ha-manel) [inline src/modules/Tasks.js]: - [x] As stated in the requirements, if a 5th task is added to the list, the index of that task should equal to 5, so counting the index should start from 1 not from 0 😅 Kindly fix this issue 👍

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @Nkaleth 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:

- [ ] Please consider using best practices when you write commit messages, they should be capitalized sentences, example: "Add linters" 👍 
- [ ] Kindly add the MIT file to your project because the link in your README file is broken 👍 

_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -194,4 +192,4 @@ I would like to thank Microverse<br>
 
 This project is [licensed](./LICENSE) .
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
\ No newline at end of file
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (rotimiazeez) [inline README.md:195]: - Kindly fix the broken License as suggested by the previous reviewer 😉
Screenshot from 2023-01-20 22-34-31

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
## Hi @Nkaleth 😉, 

# Great work on making the changes 👏 
You've done well implementing some of the requested changes, but some still need to be addressed.
### To Highlight 👇🏽
- No Linter error ✔️
- Great PR description 👍🏽

## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something needs to be 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Nkaleth , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**